packages/babel-node/test/.babelrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Rebased 696e63c58ba98e90c486b390918583c5ba59147d and was getting test errors without the `.babelrc`. 

I had to revert it back.